plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/util/api.py

```diff
@@ -229,13 +255,16 @@ def _advanced_authentication(self, api_key_id: int, api_key: str) -> dict:
         # Calculate sha256:
         api_key_hash = hashlib.sha256(auth_key).hexdigest()
         # Generate HTTP call headers
-        self.headers = {
+        return {
```

Comment:
@jmcadams-r7 I changed this to `return` from `self.headers = ` since we assign `self.headers` to the value returned from this function on L29, `self.headers = self._advanced_authentication(self.api_key_id, self.api_key)`